runtime/compilers/rillv1/parse_api.go

```diff
@@ -114,6 +127,10 @@ func (p *Parser) parseDataYAML(raw *DataYAML, contextualConnector string) (strin
 	var refs []ResourceName
 	resolverProps := make(map[string]any)
 
+	if raw.SkipResolverSecurity {
+		resolverProps["skip_resolver_security"] = raw.SkipResolverSecurity
+	}
```

Comment:
Can you make this a typed field on the `APISpec` instead? Since it's a property that should be handled upstream, not inside the individual resolver implementations.